pkg/network/ebpf/c/http-maps.h

```diff
@@ -45,4 +45,40 @@ struct bpf_map_def SEC("maps/http_batch_state") http_batch_state = {
     .namespace = "",
 };
 
+struct bpf_map_def SEC("maps/ssl_sock_by_ctx") ssl_sock_by_ctx = {
+    .type = BPF_MAP_TYPE_HASH,
+    .key_size = sizeof(void *),
+    .value_size = sizeof(ssl_sock_t),
+    .max_entries = 1024,
```

Comment:
Is `1024` enough, or should this be controlled by a configuration value?

Reply:
Humm, I don't think it needs to be controlled by a configuration param, but I think at the very least this should match the map size of `http_in_flight`. So we need to settle on a reasonable number for maximum number of *concurrent* HTTP requests.

Reply:
on second thought, nvm. I'll let `ssl_sock_by_ctx` and `http_in_flight` both be controlled by `config.MaxTrackedConnections` for the sake of consistency